### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thomashchan1**
Added GoLang and Python meeting notes doc links, plus GoLang Gitter link.

### CODE REVIEW

**Comment by thomashchan1** [comment]
Just signed CLA: You are already authorized to contribute code to this project through your membership with CNCF - LightStep.

File: README.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ Conference ID: 214 680 28#
 
 ### GoLang SDK
 
+Meeting notes are held in this [doc] (https://docs.google.com/document/d/1A63zSWX0x2CyCK_LoNhmQC4rqhLpYXJzXbEPDUQ2n6w/edit#).
```
Comment (SergeyKanzhelev) [inline README.md]:
```suggestion
Meeting notes are held in this [doc](https://docs.google.com/document/d/1A63zSWX0x2CyCK_LoNhmQC4rqhLpYXJzXbEPDUQ2n6w/edit#).
```

File: README.md [hunk 2]
```diff
@@ -70,6 +70,9 @@ Conference ID: 214 680 28#
...
+
+You can also join us on [gitter] (https://gitter.im/open-telemetry/opentelemetry-go)
```
Comment (SergeyKanzhelev) [inline README.md]:
```suggestion
You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-go)
```

File: README.md [hunk 3]
```diff
@@ -86,6 +89,8 @@ You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-
 
 The Python SIG meets weekly on [Thursday 9 AM PDT](https://calendar.google.com/calendar/embed?src=google.com_346fplg7bemb44783pege5amj4%40group.calendar.google.com&ctz=America%2FLos_Angeles). This meeting is subject to change depending on contributors' availability, check the calendar for specific dates.
 
+Meeting notes are held in this [doc] (https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit).
```
Comment (SergeyKanzhelev) [inline README.md]:
```suggestion
Meeting notes are held in this [doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit).
```

Comment (SergeyKanzhelev) [reply README.md]: this is view only access. Expected? I was approaching it by giving comment access to the links in readme and write access for the links in meeting invites.

**Comment by thomashchan1** [comment]
Thanks, @SergeyKanzhelev Done accepting suggestions. Not sure about the question on the second doc though. Believe I only modified the README.md

**Pull request merged**